packages/lesswrong/lib/routes.ts

```diff
@@ -553,6 +553,21 @@ if (isLWorAF) {
   });
 }
 
+if (isLWorAF && tagUrlBaseSetting.get() !== 'p') {
+  addRoute(
+    {
+      name: 'slashPtoBaseRedirect',
+      path: '/p/:slug',
+      redirect: (location) => {
+        const { params: { slug }, query } = location;
+        const queryString = Object.entries(query)
```

Comment:
Not the right way to do this. Not sure if it has any consequenes that matter much, since lens-slugs can't have special characters in them in practice, but this pings one of my "could be a security vulnerability" heuristic.